hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/MergeTableRegionsProcedure.java

```diff
@@ -612,9 +610,7 @@ private void createMergedRegion(final MasterProcedureEnv env) throws IOException
     List<Path> mergedFiles = new ArrayList<>();
     for (ColumnFamilyDescriptor hcd : htd.getColumnFamilies()) {
       String family = hcd.getNameAsString();
-      Configuration trackerConfig =
-        StoreFileTrackerFactory.mergeConfigurations(env.getMasterConfiguration(), htd, hcd);
-      StoreFileTracker tracker = StoreFileTrackerFactory.create(trackerConfig, family, regionFs);
+      StoreFileTracker tracker = StoreFileTrackerFactory.create(env.getMasterConfiguration(),htd, hcd, regionFs);
```

Comment:
We don't need merge master config anymore? I believe we still need this, maybe we can move this merge logic to "StoreFileTrackerFactory.create" itself, but currently `StoreFileTrackerFactory.create` is only checking the passed config instance for the store file tracker impl.

Reply:
We should call mergeConfiguations in the create method but seemed I missed it. Thank you for pointing out. Let me fix.